ssz/sedes/signed_serializable.py

```diff
@@ -0,0 +1,70 @@
+from typing import (
+    NamedTuple,
+    Optional,
+    Tuple,
+)
+
+import ssz
+from ssz.constants import (
+    SIGNATURE_FIELD_NAME,
+)
+from ssz.sedes.base import (
+    BaseSedes,
+)
+from ssz.sedes.container import (
+    Container,
+)
+from ssz.sedes.serializable import (
+    BaseSerializable,
+    MetaSerializable,
+)
+
+
+class SignedMeta(NamedTuple):
+    has_fields: bool
+    fields: Optional[Tuple[Tuple[str, BaseSedes]]]
+    container_sedes: Optional[Container]
+    signed_container_sedes: Optional[Container]
+    field_names: Optional[Tuple[str]]
+    field_attrs: Optional[Tuple[str]]
+
+
+class MetaSignedSerializable(MetaSerializable):
+
```

Comment:
whitespace? i'll let you decide if you want to patch these up, not blocking merging imo